common_repo_templates/SECURITY.md

```diff
@@ -1,3 +1,3 @@
 # Security Response
 
-If you've found a security issue that you'd like to disclose confidentially, please contact Red Hat's Product Security team. See [Security Contacts and Procedures](https://access.redhat.com/security/team/contact).
+For Security reporting and questions, please view the Open Cluster Management (SECURITY.md)[https://github.com/open-cluster-management/community/blob/main/SECURITY.md] for details.
```

Comment:
Could you see the changes in https://github.com/open-cluster-management/community/pull/33 its been reviewed and modified by the doc team so I trust that wording a bit better.

I will close that PR once you copy the changes over.